Add Basic examples that can be included in README (#64)
Citi/scaler: examples/readme.md

> ## This directory lists examples of using scaler. 

Reviewer: Succinctness.

```suggestion
## Scaler Examples
```

> If you wish to run examples in current working directory, prefix "python run_xxx.py" with "PYTHONPATH=.."  

Reviewer: ```suggestion
If you wish to run examples in current working directory, prefix `python run_xxx.py` with `PYTHONPATH=..`
```

> Make sure scheduler and cluster are set up before running clients.

Reviewer: ```suggestion
Ensure that the scheduler and cluster are set up before running clients.
```

> - disconnect_client.py 
>     Shows how to disconnect a client from scheduler 
> - graphtask_client.py
>     Shows how to send a graph based task to scheduler
> - graphtask_nested_client.py
>     Shows how to dynamically build graph in the remote end 
> - map_client.py
>     Shows how to use client.map
> - nested_client.py
>     Shows how to send a nested task to scheduler
> - simple_client.py
>     Shows how to send a basic task to scheduler
> - simple_scheduler.py
>     Shows how to initialize a Scheduler
> - simple_cluster.py
>     Shows how to initialize a Cluster

Reviewer: This is Imo, but I think file names should be preformatted.

```suggestion
- `disconnect_client.py`
    Shows how to disconnect a client from scheduler 
- `graphtask_client.py`
    Shows how to send a graph based task to scheduler
- `graphtask_nested_client.py`
    Shows how to dynamically build graph in the remote end 
- `map_client.py`
    Shows how to use client.map
- `nested_client.py`
    Shows how to send a nested task to scheduler
- `simple_client.py`
    Shows how to send a basic task to scheduler
- `simple_scheduler.py`
    Shows how to initialize a Scheduler
- `simple_cluster.py`
    Shows how to initialize a Cluster
```
